sys-cluster/pacemaker/pacemaker-2.1.2.ebuild

```diff
@@ -34,39 +34,40 @@ REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
 S="${WORKDIR}/${PN}-${MY_P}"
 
+PATCHES=( "${FILESDIR}"/${P}-command-not-found.patch )
+
 src_prepare() {
 	default
-	sed -i -e "s/ -ggdb//g" configure.ac || die
+	sed -i -e "s/ -ggdb//g" configure.ac || die "Failed to remove default gdb"
 	eautoreconf
 }
 
 src_configure() {
 	# appends lib to localstatedir automatically
-	local myconf=(
-		--with-ocfdir=/usr/$(get_libdir)/ocf
-		--localstatedir=/var
-		--disable-fatal-warnings
-		--disable-static
-		--without-cs-quorum
-		--without-cman
-		--without-heartbeat
-		--with-corosync
-		--with-ais
-		$(use_with acl)
-		$(use_with smtp esmtp)
+	econf \
```

Comment:
wait, really? Non-arrays are way more error prone (easier to miss a \) and it's generally considered cleaner to have an array :(